### PR TITLE
bypass room check for spark bot

### PR DESCRIPTION
**Pull request by tonybaloney**

### CODE REVIEW

File: scripts/stackstorm.js [hunk 1]
```diff
@@ -267,6 +267,9 @@ module.exports = function(robot) {
       room = String(msg.message.user.thread_id);
       name = msg.message.user.name[0];
     }
+    if (robot.adapterName === "spark") {
+      room = "spark" // unsupported
```
Comment (manasdk) [inline scripts/stackstorm.js]: Can you add a comment to explain this special value.

File: scripts/stackstorm.js [hunk 2]
```diff
@@ -267,6 +267,10 @@ module.exports = function(robot) {
       room = String(msg.message.user.thread_id);
       name = msg.message.user.name[0];
     }
+    if (robot.adapterName === "spark") {
+      room = String(msg.message.user.room);
```
Comment (manasdk) [inline scripts/stackstorm.js]: why does this need to cast to a String while msg.message.user.name does not

Comment (tonybaloney) [inline scripts/stackstorm.js]: ok @manasdk updated.

File: scripts/stackstorm.js [hunk 3]
```diff
@@ -267,6 +267,10 @@ module.exports = function(robot) {
       room = String(msg.message.user.thread_id);
       name = msg.message.user.name[0];
     }
+    if (robot.adapterName === "spark") {
+      room = msg.message.user.room;
+      name = msg.message.user.name
```
Comment (manasdk) [inline scripts/stackstorm.js]: also missing ;

**Pull request merged**